src/controllers/bill.js

```diff
@@ -113,3 +287,23 @@ module.exports.getBillStatusOfMembersInASession = async (req, res) => {
     return res.send({ message: err.message });
   }
 };
+
+
+const updateNewPartialTransactions=(partialTransactions,billData,newPartialTransactions)=>{
```

Comment:
this should return some value instead of doing mutation to the newPartialTransaction.